resources/assets/components/UploaderModal/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReportbackUploader from '../ReportbackUploader';
+
+class UploaderModal extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    const signup = this.props.signup;
+    const campaign = this.props.campaign;
+
+    const photoUploaderProps = {
+      campaignId: signup.campaign_id,
+      campaignRunId: signup.campaign_run_id,
+      northstarId: signup.northstar_id,
+      noun: {
+        plural: campaign.reportback_info.noun,
+      },
+      // @TODO: do we want this to be the admin's northstar ID so we know who submitted this?
+      source: 'Rogue admin',
```

Comment:
should we have this be more specific? Maybe something like "Rogue admin: [northstar id] or [name]"? Or is this ok and we can always check the logs for this?

Reply:
hmm we store the user in the events table, right? If so, then I don't think we need it here, but I think the source should be formatted `rogue-admin` for easier parsing.

Reply:
Yes we do! Oh great call - I'll change that now!